libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -19,6 +19,10 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * @file mqtt_lightweight.c
+ * @brief Implements the user facing functions in mqtt_lightweight.h.
```

Comment:
Nit: consistency with header file
```suggestion
 * @brief Implements the user-facing functions in mqtt_lightweight.h.
```

Reply:
sure